R/data.R

```diff
@@ -68,9 +68,9 @@
 #' ep_companies
 "ep_companies"
 
-#' isic-tilt mapper
+#' ISIC
 #' @author Kalash Singhal.
 #'
 #' @examples
-#' isic_tilt_mapper
-"isic_tilt_mapper"
+#' isic
+"isic"
```

Comment:
@maurolepore I think the new name for `isic_tilt_mapper` should be `isic_name` because the new file contains only two columns `isic_4digit` and `isic_4digit_name` which will be used to assign `isic_4digit_name` column to the final outputs using the column `isic_4digit`. 

`isic` could be used to rename the column `isic_4digit` in the future, hence I request you to change the name of `isic_tilt_mapper` to `isic_name`. Do you agree?

Reply:
Thanks @kalashsinghal, the dataset is now called `isic_name`. You can see the updated reprex in the top comment.